reference-architecture/aws-ansible/playbooks/roles/rhsm-repos/tasks/main.yaml

```diff
@@ -8,3 +8,12 @@
       - rhel-7-server-rpms
       - rhel-7-server-extras-rpms
       - rhel-7-server-ose-3.3-rpms
+
+- name: remove rhui repos
+  file:
+    path: "{{ item }}"
+    state: absent
+  with_items:
+      - /etc/yum.repos.d/redhat-rhui-client-config.repo
+      - /etc/yum.repos.d/redhat-rhui.repo
+      - /etc/yum.repos.d/rhui-load-balancers.conf
```

Comment:
rhui killed all of my playbook runs this morning due to a missing CDS. I could always do a regex or replace to disable the repo in the file. It doesn't look like yum-config-manager bites in disabling these repos sadly.

```
[root@ip-10-20-4-228 ~]# yum-config-manager --disable rhui-REGION-rhel-server-releases/7Server/x86_64
Loaded plugins: amazon-id, rhui-lb
[root@ip-10-20-4-228 ~]# yum-config-manager --disable rhui-REGION-rhel-server-rh-common/7Server/x86_64
Loaded plugins: amazon-id, rhui-lb
[root@ip-10-20-4-228 ~]# yum repolist
Loaded plugins: amazon-id, rhui-lb, search-disabled-repos, subscription-manager
Could not contact CDS load balancer rhui2-cds01.us-east-1.aws.ce.redhat.com, trying others.
repo id                                                                                        repo name                                                                                                     status
rhel-7-server-extras-rpms/x86_64                                                               Red Hat Enterprise Linux 7 Server - Extras (RPMs)                                                                297
rhel-7-server-ose-3.3-rpms/x86_64                                                              Red Hat OpenShift Container Platform 3.3 (RPMs)                                                                  542
rhel-7-server-rpms/7Server/x86_64                                                              Red Hat Enterprise Linux 7 Server (RPMs)                                                                      11,396
rhui-REGION-client-config-server-7/x86_64                                                      Red Hat Update Infrastructure 2.0 Client Configuration Server 7                                                    6
rhui-REGION-rhel-server-releases/7Server/x86_64                                                Red Hat Enterprise Linux Server 7 (RPMs)                                                                      11,396
rhui-REGION-rhel-server-rh-common/7Server/x86_64                                               Red Hat Enterprise Linux Server 7 RH Common (RPMs)                                                               209
repolist: 23,846
[root@ip-10-20-4-228 ~]# yum install zsh
Loaded plugins: amazon-id, rhui-lb, search-disabled-repos, subscription-manager
Could not contact CDS load balancer rhui2-cds01.us-east-1.aws.ce.redhat.com, trying others.
```